File: DE_1/contestants/nguyen_duy_hung/bai03.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
    ifstream inp("bai03.inp");
    ofstream out("bai03.out");
    int n,a;
    inp >> n;
    vector<int> A;
    while(n--) {
        inp >> a;
        A.push_back(a);
    }
    int s=0;
    for(int i=0; i<A.size(); i++) {
        vector<int> B=A;
        B.erase(B.begin()+i);
        for(int j=0; j<B.size(); j++) {
            for(int k=j; k<B.size(); k++) {
                if(A[i]*2==B[j]+B[k]) {
                    s++;
                    j=A.size()-1;
                }
            }
        }
    }
    out << s;
    out.close();
    return 0;
}

File: DE_1/contestants/bach_do/bai03.py
with open("bai03.inp","r") as fi:
    k=int(fi.readline())
    n=list(map(int,fi.readline().split()))
    d=0
    for i in n:
        for j in n:
            if i%j==0 and i!=j:
                d+=1
            break
with open("bai03.out","w") as fo:
    fo.write(str(d-1))
fo.close()
fi.close()
